webapp/pages/registration/signup.vue

```diff
@@ -22,7 +22,7 @@ export default {
   },
   asyncData({ app }) {
     return {
-      publicRegistration: app.$env.PUBLIC_REGISTRATION === 'true',
+      publicRegistration: app.$env.PUBLIC_REGISTRATION,
```

Comment:
@Mogge Why you using `app.$env` here if you seem to prefer `this.$env`?
… like you said here https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/4292#issuecomment-801803624

I don’t know the differences …